src/styles/utils/variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$blue-accent-color: #00acdc;
```

Comment:
Nice! Would be great to create variables for all colors and other main values (card-width, padding etc.)